_episodes/06-func.md

```diff
@@ -409,7 +420,8 @@ TypeError                                 Traceback (most recent call last)
 <ipython-input-26-e3bc6cf4fd6a> in <module>()
 ----> 1 numpy.loadtxt('inflammation-01.csv', ',')
 
-/Users/gwilson/anaconda/lib/python2.7/site-packages/numpy/lib/npyio.pyc in loadtxt(fname, dtype, comments, delimiter, converters, skiprows, usecols, unpack, ndmin)
+/Users/gwilson/anaconda/lib/python2.7/site-packages/numpy/lib/npyio.pyc 
```

Comment:
Let's not break the line here because this is output. We'll have to modify scripts not to complain about long lines in the `output` section.